src/rpc.rs

```diff
@@ -1,15 +1,20 @@
-#[derive(PartialEq, Clone, Debug, Copy)]
+#[derive(PartialEq, Clone, Debug)]
 #[repr(C)]
 pub struct Rpc {
    pub data    : u32,
    pub uid     : u64,
+   pub path    : String,
```

Comment:
Can we add inline comments saying what data, uid, and path mean?

Reply:
Good idea!  Should be done.